code/testapp/src/main/java/com/adobe/mobile/marketing/userprofile/testapp/MyApp.kt

```diff
@@ -25,7 +26,8 @@ class MyApp : Application() {
         MobileCore.registerExtensions(
             listOf(
                 UserProfile.EXTENSION,
-                Signal.EXTENSION
+                Signal.EXTENSION,
```

Comment:
I don't think we need to include Signal extension in UserProfile test app. Can you remove the extension from registration and gradle file?